### PR TITLE
Make diving goggles visible in dive shop

### LINKED ISSUE #226: Diving goggles not showing up in clothing menu
### Expected behaviour
If you go to the clothing shop menu, you should be able to see diving googgles.

### Actual behaviour
You don't see them, because the player is not underwater.

### Steps to reproduce the behaviour 
Open any shop wich sells diving goggles and find out by yourself!

### [RPT](https://community.bistudio.com/wiki/Crash_Files#Arma_3) and/or extDB2 logs (if applicable)
NO RPT (not an actual error)

----

Mission version: ALL

Launcher version: ALL

Game version: ALL

Branch: What's a branch?

### PR DESCRIPTION
**Pull request by Blauschimmel**
Resolves #226

### CODE REVIEW

File: Altis_Life.Altis/core/shops/fn_clothingMenu.sqf [hunk 1]
```diff
@@ -133,6 +133,10 @@ if (LIFE_SETTINGS(getNumber,"clothing_noTP") isEqualTo 1) then {
     };
 
     player setBehaviour "SAFE";
+    if(_shop == "dive") then {
```
Comment (NeFShARk) [inline Altis_Life.Altis/core/shops/fn_clothingMenu.sqf]: Missing a white space after the **if**.
And i think you should change **==** to  **isEqualTo**.

Comment (Blauschimmel) [inline Altis_Life.Altis/core/shops/fn_clothingMenu.sqf]: the == is also used in line 61, still sure to change it?

**Comment by WolfSkin1337** [comment]
Use spacebars Not tab. 
1 tab = 4 spacebars

**Comment by Blauschimmel** [comment]
I see i see

**Comment by Blauschimmel** [comment]
Are you sure to change "==" to "isEqualTo"? "==" is used in line 61 as well and makes the code shorter... Or does it have an other effects?

**Comment by NeFShARk** [comment]
@Blauschimmel here - https://community.bistudio.com/wiki/isEqualTo

**Comment by Jason2605** [comment]
Yea since you are changing it, can you also change the == to isEqualTo

**Comment by NeFShARk** [comment]
Yeah, just change the == on lines 59, 60 and 61 to isEqualTo.

File: Altis_Life.Altis/core/shops/fn_clothingMenu.sqf (hunk 1 above)
Comment (NeFShARk) [inline Altis_Life.Altis/core/shops/fn_clothingMenu.sqf]: Yes.

**Comment by Blauschimmel** [comment]
So I'm going to change them all back again? I'm just waiting for a go...

**Comment by Blauschimmel** [comment]
@Jawshy should I?

**Comment by Jawshy** [comment]
**If** `==` works fine then you should not have to change it.

**Comment by Blauschimmel** [comment]
I already changed all to isEqualTo... so I'll just undo it!

**Comment by BoGuu** [comment]
So this works? Never noticed the issue before. Can you see the teleport to water?

**Comment by Jawshy** [comment]
@BoGuu, yes to the first question and no to the second.

**Comment by Blauschimmel** [comment]
@BoGuu, you'll be able to see one frame underwater if your server isn't very performative or you got a huge fps rate. But it wouldn't be long enough to realise it.

**Pull request merged**